src/plugins/intel_cpu/src/nodes/common/cpu_convert.cpp

```diff
@@ -467,6 +468,26 @@ bool isConversionTruncatesRange(const Precision & from, const Precision & to) {
             || (to == Precision::BOOL && from != to);   // T -> bool
 }
 
+struct MKLDNNCopyContext {
+        const void *srcPtr;
+        void *dstPtr;
+        size_t size;
+        Precision dstPrc;
+        bool converted;
+    };
+
+template<typename ele_t>
+struct CopyElement {
+    void operator()(MKLDNNCopyContext &ctx) {
```

Comment:
can we perform coping as data agnostic? one thread will copy `byte_size / max_threads_num` bytes? you can use `parallel_nt` for this purpose I guess

Reply:
Thanks a lot, change to type-agnostic parallel copy in 6fa2ec224a63001130eabfae3094c95dbc4eb9aa